orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/kato/tasks/quip/InstanceHealthCheckTaskSpec.groovy

```diff
@@ -90,6 +90,10 @@ class InstanceHealthCheckTaskSpec extends Specification {
     def stage = new Stage(pipe, 'instanceHealthCheck', [:])
     stage.context.instances = instances
 
+    and:
+    task.createInstanceService(_) >> instanceService
+    instanceService.healthCheck("healthCheck") >> new Response('http://foo.com', 200, 'OK', [], new TypedString("Good"))
```

Comment:
Without this check we were making real connections to `http://foo.com:7001/healthCheck` which takes 10-20 seconds to fail. Also, that's pretty obnoxious.

Reply:
lol ok